recipes-openxt/xenclient-toolstack/xenclient-toolstack_git.bb

```diff
@@ -2,8 +2,8 @@ DESCRIPTION = "XenClient toolstack"
 LICENSE = "LGPLv2.1"
 LIC_FILES_CHKSUM = "file://COPYING;md5=321bf41f280cf805086dd5a720b37785"
 DEPENDS += "ocaml-dbus camomile xen xz"
-RDEPENDS_${PN} = "xen-xenstore xen-xenstored"
-RDEPENDS_${PN}_xenclient-ndvm += " dbd-tools-vm"
+RDEPENDS_${PN} = "xen-tools-xenstore virtual/xenstored"
+RDEPENDS_${PN}_xenclient-ndvm += " db-tools-vm"
```

Comment:
`db-tools-vm` is `dbd-tools-vm` since https://github.com/OpenXT/xenclient-oe/pull/1321, seems like a rebase induced error.

Reply:
ack, have refreshed the branch to fix this